### DISCUSSION TITLE
Official or not?

### DISCUSSION THREAD

**Discussion by dmki** [betalgo/openai#260] [General]
My understanding is, that this package is not a product of OpenAI company. Hence the word "Unofficial" at Github page.

However, the nuget package claims that this package's project URL is "https://openai.com", which can only be true if it's official project. Otherwise it should lead to Github page or whatever is the master page of this project.

Could you please sync information between the GitHub and NuGet?

**Comment by kayhantolga** [comment]
The library clearly mentions that this is not an official SDK, and OpenAI does not have any official .Net SDK. However, it is one of the community libraries available, as described on the [OpenAI website](https://platform.openai.com/docs/libraries/community-libraries).

The project website URL has been set to the OpenAI website to provide more information, and the source repository URL can be followed for those who need additional information.

**Reply by dmki** [reply]
Yes, and that's incorrect. That link should lead to the page of YOUR project, where people could get information about your package, not about what OpenAI or ChatGPT are. Look at other packages and how they are decorated.
In worst case scenario you'll get C&D from OpenAI. Plus, if other packages don't lead to OpenAI, but yours does, it may be considered as unfair competition.

When someone needs to find information about the package, like version history etc, they first go to NuGet Report Abuse page, and from there - to this github page, for all other packages there is one simple direct link. See the problem?
